app/views/datasets/show.html.erb

```diff
@@ -2,7 +2,21 @@
   Check your dataset -
 <% end %>
 
-<h1 class="heading-large">Publish ‘<%= @dataset.title %>’</h1>
+<% if @dataset.errors.any? %>
+<div class="error-summary" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+    <h1 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+      There was a problem
+    </h1>
+
+    <ul class="error-summary-list">
+			<% @dataset.errors.each do |attr, error| %>
```

Comment:
Indentation's all messed up, and there are a few markup issues. But I'll be reviewing all templates as part of the corresponding Trello story